styles.css

```
@@ -0,0 +1,25 @@
+body {
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
```

I believe leaving an empty line between classes would make the file more readable.
```suggestion
  }
  
```